src/schema/agency-budget.ts

```diff
@@ -4,8 +4,10 @@ import { agency } from "./agency";
 
 export const agencyBudget = pgTable("agency_budget", {
   code: text("code").primaryKey(),
-  type: text("type"),
-  sponsor: text("sponsor").references(() => agency.initials),
+  type: text("type").notNull(),
```

Comment:
Please generate a migration to apply these "pgTable" schema updates to the database